app/models/user.rb

```diff
@@ -10,6 +10,10 @@ class User < ApplicationRecord
             uniqueness: { case_sensitive: false }
 
   has_many :notifications, dependent: :destroy
+  has_many :deposits, class_name: 'Work',
+                      foreign_key: 'depositor_id',
+                      inverse_of: :depositor,
+                      dependent: :destroy
```

Comment:
Does this mean if we destroy a user instance, we lose all their works?

Reply:
Yes.

Reply:
We'll have db backups if we need them, I reckon.